net-misc/sipp/sipp-3.6.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="SIPp is a free Open Source test tool / traffic generator for the SIP protocol"
+HOMEPAGE="http://sipp.sourceforge.net/ https://github.com/SIPp/sipp/releases"
+SRC_URI="https://github.com/SIPp/sipp/releases/download/v${PV}/${PF}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl +pcap sctp +ssl"
+
+DEPEND="sys-libs/ncurses:=
+	ssl? ( dev-libs/openssl:= )
+	pcap? ( net-libs/libpcap net-libs/libnet:1.1 )
+	gsl? ( sci-libs/gsl:= )
+"
+RDEPEND="${DEPEND}"
+
+src_configure()
+{
+	econf \
+		$(use_with gsl) \
+		$(use_with pcap) \
+		$(use_with sctp) \
+		$(use_with ssl openssl) \
+		--with-rtpstream
```

Comment:
You could use the array form, but up to you really to decide.

Reply:
I'm not sure what you mean with "array form".  I like how it's written, but if there is some array form that may this read even simpler I'd like to look at that.

Can't seem to find any reference in ebuild(5) to an array form for this.

Reply:
```
	local myeconfargs=(
	        $(use_with gsl)
	        $(use_with pcap)
	        $(use_with sctp)
	        $(use_with ssl openssl)
	        --with-rtpstream

	econf "${myeconfargs[@]}"
```

I've heard backlashes can cause this and that, but it's really up to your preference in the end.

Reply:
I think I'll stick with what's there currently.

\ is standard shell. () is bash specific.  Not that I think this will ever use anything non-bash.